app/assets/stylesheets/base/_form.scss

```diff
@@ -11,3 +11,9 @@
     }
   }
 }
+
+.employment-gap-buttons {
```

Comment:
Suggestion for consistency. I think styles over a particular element are named in singular: `.employment-gap-button`